### DISCUSSION TITLE
How can I do synteny using multiple gff files?

### DISCUSSION THREAD

**Discussion by paularf** [moshi4/pyGenomeViz#33] [Q&A]
Good day:
 I am trying to do synteny with gff files from different genomes and while the tutorials are very good for doing this with genebank files (eg., Practical  example 3. Plot Genbank Files & Links in https://moshi4.github.io/pyGenomeViz/api-examples/practical_example/#3-plot-genbank-files-links). I couldn't find a tutorial to do it using gff files as input. Do you think you can help me? for now I only have been capable of plotting a single gff file, but I want to do a figure of synteny using multiple gff files.

Thank you very much in advance, I would really appreciate your help.

Paula

**Comment by moshi4** [comment]
There is no way to create synteny plots from GFF files in pyGenomeViz. pyGenomeViz only provides a way to perform a simple pairwise genome alignment between multiple Genbank files and visualize the results.

**Reply by paularf** [reply]
I see, thank you anyways!
